### ISSUE TITLE
A11y_AzureDataStudio_Welcome_ExtendyourdataStudio_ColorContrast : The color contrast ratio of "Show All" control is less than 4.5:1.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/azuredatastudio#11549]
**"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**
GitHubTags:#A11y_AzureDataStudioJuly2020;#A11y_July27_2020_TestPass;#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#MAC;#Screenreader;#ColorContrast;#A11ySev2;#Benchmark;#MAS1.4.3;#FTP;

### Environment Details:
Application Name: Azure Data Studio
Application Version: 1.21.0-insider
Commit: eccf3cf
Date: 2020-07-24T09:28:31.172Z
VS Code: 1.48.0
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina (Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.4.3

### Repro Steps:
1. Launch Azure Data Studio Insiders application.
2. If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3. Navigate Extend your data Studio on Welcome screen.
4. Navigate to the "Show All" control then open Colour Contrast Analyser tool and check if the ratio is more than or equal to 4.5:1.

### Actual:
The color contrast ratio of the "Show All" control is 1.8:1 which is less then the standard color contrast ratio of 4.5:1 due to which this control will be hard to read for low vision users.

### Expected:
The color contrast ratio of "Show All" control should be more than or equal to 4.5:1.

### User Impact:
If the content present on the screen has color contrast ratio less than 4.5:1 the it will be hard for low vision users to read the content on the screen and they can miss out on the information provided.

### Attachment link for Reference:
[11549_A11y_AzureDataStudio_Welcome_ExtendyourdataStudio_ColorContrast The color contrast ratio of "Show All" control is less than 4.5 1..zip](https://github.com/microsoft/azuredatastudio/files/4985939/11549_A11y_AzureDataStudio_Welcome_ExtendyourdataStudio_ColorContrast.The.color.contrast.ratio.of.Show.All.control.is.less.than.4.5.1.zip)

**Comment by Raghu-addepalli** [comment]
Issue is fixed Hence closing the bug .
Githubtags:  #Closed;
